src/pages/MerchantVoucherDashboard/MerchantVoucherDashboard.tsx

```diff
@@ -120,19 +149,28 @@ const MerchantVoucherDashboard = () => {
                 Active Vouchers{' '}
                 <span className={styles.noBreak}>可使用的礼品券数量</span>
               </h1>
-              <h2>{giftCards && giftCards.length}</h2>
+              <h2>{giftCards && giftCards.filter(card => card.latest_value > 0).length}</h2>
             </div>
             <div className={styles.metadataBlock}>
               <h1>
                 Total Balance <span className={styles.noBreak}>总结余</span>
               </h1>
               <h2>
                 {renderAmount({
-                  value: giftCards.reduce((acc, cur) => acc + cur.value, 0),
+                  value: giftCards.reduce((acc, cur) => acc + cur.latest_value, 0),
                 })}
               </h2>
             </div>
           </div>
+          <div className={styles.gamToggle}>
```

Comment:
.

Reply:
@aliceluo2048, replying here to keep it all together. I can't change the word "filter" unfortunately, as it's fixed within the component I'm importing. It does "filter" the records but it's basically a search. Something to do in V2!